_data/tutorials.yaml

```diff
@@ -1525,3 +1527,16 @@ survey-responses:
     responses happen.'
   status:
     confluent: disabled
+versioned-ktables:
+  title: Versioned KTables for temporal join accuracy
+  meta-description: Ensure proper stream-table temporal join semantics using a versioned statestore to back your KTable
+  canonical: confluent
+  slug: /versioned-ktables
+  question: How can you ensure your stream-table join results are as correct as possible?
+  introduction: It used to be when Kafka Streams executes a stream-table join the stream side event would join the the latest avaialble
+      record with the same key on the table side.  But sometimes it's important for the stream event to match up with a table record by timestamp
+      as well as key, think of a stream of stock transactions and a table of stock prices - it's essential the transaction joins with the stock price
```

Comment:
suggest two sentences, and use the long em dash rather than dash in the second:

> But sometimes it's important for the stream event to match up with a table record by timestamp
      as well as key. Consider a stream of stock transactions and a table of stock prices — it's essential the transaction joins with the stock price at the time of the transaction,